webapp/packages/core-blocks/src/useControlledScroll.test.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { beforeEach, describe, expect, it, jest } from '@jest/globals';
+import { renderHook } from '@testing-library/react';
+
+import { IScrollState, useControlledScroll } from './useControlledScroll';
+
+describe('useControlledScroll', () => {
+  let element: HTMLDivElement;
+  let scrollState: IScrollState;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    jest.useFakeTimers();
+    element = document.createElement('div');
```

Comment:
is this element destroyed properly after each test?

Reply:
it kills the dom after all test cases in the file were executed
but after each test case it does not remove element from the dom. so I modified the test so it will remove this div element. thanks